src/Microsoft.Extensions.Logging.Console/ConsoleLogger.cs

```diff
@@ -255,14 +249,14 @@ private ConsoleColors GetLogLevelConsoleColors(LogLevel logLevel)
             }
         }
 
-        private string GetScopeInformation()
+        private void GetScopeInformation(StringBuilder builder)
```

Comment:
Not sure how much this would save, but since you are measuring perf, you could probably optimize here to avoid calculating the scope if its already calculated. (for reference...Long time back I had a PR for this change https://github.com/aspnet/Logging/pull/404/files)